launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -73,6 +74,21 @@
   /** Logger name to use when launching a child process. */
   public static final String CHILD_PROCESS_LOGGER_NAME = "spark.launcher.childProcLoggerName";
 
+  /** Launcher Server Port to use when launching a child process. */
+  public static final String LAUNCHER_INTERNAL_PORT = "spark.launcher.internal.port";
+
+  /** Launcher Server sets this when launching a child process. */
+  public static final String CHILD_PROCESS_LAUNCHER_INTERNAL_SECRET = "spark.launcher.internal.secret";
+
+  /** Stop Flag if interrupted Launcher Server goes away. */
```

Comment:
can we clarify this like mentioned below.  tries to kill the actual running application